src/benchmark/CMakeLists.txt

```diff
@@ -26,10 +26,20 @@ target_link_libraries(
     benchmark
 )
 
+# General purpose benchmark runner
+add_executable(hyriseBenchmarkRunner benchmark_runner_main.cpp)
+target_link_libraries(
+    hyriseBenchmarkRunner
```

Comment:
Together with hyriseBenchmark and hyriseBenchmarkTPCH, this gets confusing. I don't even know why we have the `hyrise` prefix. @mrzzzrm?

Maybe `./microBenchmarks`, `./benchmark` and `./benchmarkTPCH`?

Reply:
I added the `hyrise*` prefix because, e.g., there is already a target called "benchmark" - the google benchmark lib.

Reply:
I don't like the name "hyriseBenchmarkRunner" much, either. Just hyriseBenchmark would do it for me

Reply:
suggestion: `hyriseOperatorBenchmark`, `hyriseBenchmark`, `hyriseBenchmarkTPCH`